docs/Documentation/UserGuideV0.7.0/7-Tools-Sync.md

```diff
@@ -0,0 +1,278 @@
+
+The sync tool has a many-to-one sender-receiver mode, that is, one sync receiver can receive data from multiple sync senders at the same time, and one sync sender can only send data back to one sync server.
+
+> Note: Before using the sync tool, the client and server need to be configured separately. The configuration is detailed in Sections Configuration.
+# Configuration
+## Sync Receiver
+The parameter configuration of the sync receiver is located in the configuration file `iotdb-engine.properties` of IoTDB, and its directory is `$IOTDB_HOME/conf/iotdb-engine.properties`. In this configuration file, there are four parameters related to the sync receiver. The configuration instructions are as follows:
+
+<table>
+   <tr>
+      <td colspan="2">parameter: is_sync_enable</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>Sync function switch, which is configured as true to indicate that the receiver is allowed to receive the data from sender and load it. When set to false, it means that the receiver is not allowed to receive the data from any sender.</td>
+   </tr>
+   <tr>
+      <td>Type</td>
+      <td>Boolean</td>
+   </tr>
+   <tr>
+      <td>Default</td>
+      <td>false</td>
+   </tr>
+   <tr>
+      <td>Modalities for Entry into Force after Modification</td>
+      <td>Restart receiver</td>
+   </tr>
+</table>
+
+<table>
+   <tr>
+      <td colspan="2">parameter: IP_white_list</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>Set up a white list of sender IP addresses, which is expressed in the form of network segments and separated by commas between multiple network segments. When the sender transfers data to the receiver, only when the IP address of the sender is within the network segment set by the whitelist can the receiver allow the sync operation. If the whitelist is empty, the receiver does not allow any sender to sync data. The default receiver accepts all IP sync requests.</td>
+   </tr>
+   <tr>
+      <td>Type</td>
+      <td>String</td>
+   </tr>
+   <tr>
+      <td>Default</td>
+      <td>0.0.0.0/0</td>
+   </tr>
+   <tr>
+      <td>Modalities for Entry into Force after Modification</td>
+      <td>Restart receiver</td>
+   </tr>
+</table>
+
+<table>
+   <tr>
+      <td colspan="2">parameter: update_historical_data_possibility</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>The processing strategy chosen by the sync receiver when merging the sync data.<br/>
+        1. If the sync data accounts for more than 50% of the update of the historical data (compared with the latest timestamp of the local storage group data),then it is recommended to select strategy. Setting the parameter to true, which has a greater impact on the write performance of the IoTDB system and occupies less CPU of the machine.<br/>
```

Comment:
Actually, if it's set to true, it will reduce write performance. What's more, sync tool will both reduce write performance in both modes. The difference is that one mode affects CPU more, and the other mode that affects write performance more.